test/plugins/features/movies/index.test.js

```diff
@@ -18,7 +18,115 @@ describe('movies integration', () => {
       expect(response.statusCode).to.eql(200);
       expect(response.result.object).to.eql('movie');
       expect(response.result.title).to.eql(payload.title);
-      expect(response.result.release_year).to.be.null;
+      expect(response.result.release_year).to.be.a('null');
```

Comment:
What was the reason for changing this?

Reply:
![image](https://user-images.githubusercontent.com/1016325/55642203-b118c180-5784-11e9-969e-dff59d7af44a.png)